dgl_ptm/tests/test_model.py

```diff
@@ -195,13 +199,13 @@ def test_model_init_savestate_not_default(self, model):
     def test_model_init_restart(self, model):
         model.step_target = 3 # only run the model till step 3
```

Comment:
```suggestion
        model.savestate = 1
        model.step_target = 3 # only run the model till step 3
```

Reply:
In the `main` branch, the default value of `savestate` is 1. But it seems that in `development` branch is 10.

Reply:
The reason for failing test is that the function `test_model_init_savestate(self, model):` still includes `assert Path('my_model/model_data.bin').exists()`. Can you please remove that line?